dashboard/src/components/DeploymentStatus/DeploymentStatus.tsx

```diff
@@ -44,6 +49,14 @@ class DeploymentStatus extends React.Component<IDeploymentStatusProps> {
       return true;
     }
   }
+
+  private renderDeletedStatus() {
+    return (
+      <span className="DeploymentStatus DeploymentStatus--deleted">
+        <Warning className="icon padding-t-tiny" /> Deleted (Not purged)
```

Comment:
I'm not sure about the "(not purged)", I think we can make that clear by calling the delete button "Purge"